src/services/MeetingService.js

```diff
@@ -1,38 +1,41 @@
+/* eslint-disable no-await-in-loop,no-restricted-syntax,no-underscore-dangle */
 const inversify = require('inversify');
 const { TYPES } = require('../constants');
 const BaseService = require('./BaseService');
+const { AppError } = require('../middlewares/ErrorHandlers');
 
 class MeetingService extends BaseService {
   constructor(repository, userMeetingService) {
     super(repository);
     this.userMeetingService = userMeetingService;
   }
 
-  // checkIdInDb(req) {
-  //   return this.repository.findOne(req.params.id);
-  // }
-  //
-  // async checkInDb(req) {
-  //   const errors = [];
-  //
-  //   const properties = Object.getOwnPropertyNames(req.body);
-  //
-  //   for (const val of properties) {
-  //     if (val.includes('Id')) {
-  //       let newKey = val.replace('Id', '');
-  //
-  //       if (newKey.includes('creator')) {
-  //         newKey = newKey.replace('creator', 'user');
-  //       }
-  //       const manager = this.repository.manager;
-  //       const result = await manager.findOne(newKey, req.body[val]);
-  //       if (result === undefined) {
-  //         errors.push(`cannot find ${newKey}Id with id:${req.body[val]} in DB`);
-  //       }
-  //     }
-  //   }
-  //   return errors;
-  // }
+  async _checkIdInDb(id) {
+    const result = await this.repository.findOne(id);
+    if (!result) {
+      throw new AppError(`cannot find meeting with id:${id}`, 404);
+    }
+  }
+
+  async _checkPropsInDb(req) {
```

Comment:
What is the purpose of this function?

Reply:
This one is from old version. It must be removed.